### PR TITLE
fix(typo): Skadenga Saga tweaks, Bulk Freighter capitalization, and various typos fixes

### PR DESCRIPTION
**Pull request by Terin**
This PR makes various tweaks to the Skadenga Saga missions, such as re-arranging mission attributes, having auto-accepting dialogs as conversations, adding `landing` to certain missions that needed them, and other typo fixes broadly. It also capitalizes Bulk Freighter throughout ES's data, fitting in with the various outfit capitalization changes of late.

### CODE REVIEW

**Review by EjoThims — APPROVED** [review]
Changes all check out as far as I can see :D

**Review by LorenzoBolla — APPROVED** [review]
Same here, everything seems in order!

File: data/human/deep missions.txt [hunk 1]
```diff
@@ -4042,10 +4045,10 @@ mission "Hjlod Remembers Windblain"
 		has "Hjlodsaved"
 	on offer
 		dialog
-			`Your inbox hums with a new message from <planet>:`
-			`	Dear <first>,`
+			`"Your inbox hums with a new message from <planet>:`
```
Comment (Ferociousfeind) [inline data/human/deep missions.txt:4048]: Should there be a quotation mark here? This seems like a typo, the quote begins one line below (where there is another opening quotation mark)

File: data/human/deep missions.txt [hunk 2]
```diff
@@ -3374,10 +3373,11 @@ mission "Mani Refit"
 		not planet "Mani"
 	destination "Mani"
 	on offer
-		dialog
+		conversation
 			`Your inbox blinks with a new message from the Transport Union:`
-			`	The fleet is ready for its inspirational journey. Please return to Mani.`
-			`		-Jayden, Mani Administrator, Transport Union`
+			`	"The fleet is ready for its inspirational journey. Please return to Mani.`
+			`		-Jayden, Mani Administrator, Transport Union"`
```
Comment (warp-core) [inline data/human/deep missions.txt:3379]: I don't think stuff like this usually gets quotes.
See the messages the player receives in "FW Commitment" and "FW Wolf Pack 2".
You could, however, instead present the message in-line, as is done in "Smuggler's Den: Payment".

Comment (EjoThims) [reply data/human/deep missions.txt:3379]: Those others should probably be adjusted as well then (either here or separately), the addition of quotes here is correct.

If we were -actually- presenting an image of the message as a cutscene instead (which I would not support stylistically, even if it didn't sound like a whole chore in itself), then it'd be proper to leave them off, but even with using spacing to mirror that concept, it's still quoting the message, not showing it.

File: data/human/deep missions.txt (hunk 1 above)
Comment (EjoThims) [inline data/human/deep missions.txt:4048]:
```suggestion
			`Your inbox hums with a new message from <planet>:`
```

Yea, I didn't catch that loose one here, or the same swap in the next set like it.

File: data/human/deep missions.txt [hunk 3]
```diff
@@ -4071,20 +4075,20 @@ mission "Hjlod Remembers Nifel"
 		has "Hjlodsaved"
 	on offer
 		dialog
-			`Your inbox hums with a new message from Nifel.`
+			`"Your inbox hums with a new message from Nifel.`
```
Comment (EjoThims) [inline data/human/deep missions.txt:4078]:
```suggestion
			`Your inbox hums with a new message from Nifel.`
```

File: data/human/deep missions.txt [hunk 4]
```diff
@@ -4071,20 +4075,20 @@ mission "Hjlod Remembers Nifel"
...
 			`	Dear <first>,`
```
Comment (EjoThims) [inline data/human/deep missions.txt:4079]: And added here as above. Can't suggest on untouched line from my phone though.

**Review by EjoThims — CHANGES_REQUESTED** [review]

**Review by EjoThims — APPROVED** [review]

**Pull request merged**